### PR TITLE
Avoid broken markdown lint checker action version

### PR DESCRIPTION
**Pull request by bpkroth**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -341,10 +341,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+    - uses: gaurav-nelson/github-action-markdown-link-check@1.0.8
```
Comment (bpkroth) [inline .github/workflows/main.yml:344]: The `v1` tag is a meta version that pulls the latest changes for the v1 release.
This upstream change (in the 1.0.9 release) adds quotes around the "folder-path" provided below which causes `find` to fail.
https://github.com/gaurav-nelson/github-action-markdown-link-check/commit/53c1dda67d2de0a25bbdce0b1a4054cfbb468b32

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -341,10 +341,11 @@ jobs:
...
       with:
         use-quiet-mode: yes
-        folder-path: './documentation, ./build, ./source, ./test, ./scripts'
+        use-verbose-mode: no
+        folder-path: './documentation, ./build, ./source, ./test, ./scripts, ./external'
```
Comment (bpkroth) [inline .github/workflows/main.yml:348]: Adding the `./external` path to the set of things to check.

**Review by byte-sculptor — APPROVED** [review]

**Comment by bpkroth** [comment]
Upstream fix:
https://github.com/gaurav-nelson/github-action-markdown-link-check/pull/88

**Comment by bpkroth** [comment]
https://github.com/gaurav-nelson/github-action-markdown-link-check/releases/tag/v1
(or 1.10) now includes my fix, so we can probably revert the version change part of this.

**Pull request merged**